src/Admin/AbstractAdmin.php

```diff
@@ -2837,7 +2839,14 @@ protected function buildForm()
             }
         }
 
-        $this->form = $this->getFormBuilder()->getForm();
+        $formBuilder = $this->getFormBuilder();
+        $formBuilder->addEventListener(FormEvents::POST_SUBMIT, function (FormEvent $event) {
+            if (method_exists($this, 'preValidate')) {
```

Comment:
Do you need this check? The method will always exist in an `AbstractAdmin`

Reply:
Not if it is not on the interface

Reply:
oh, I didnt saw that this method is directly on the abstractAdmin, so this is indeed not needed. Sorry for the mistake, the if is not needed. Can you remove it?